quick-guide-for-senior-developers.md

```diff
@@ -0,0 +1,80 @@
+
+# The Way We Work: Quick Guide for Senior Developers
+
+## Your Role
+
+Your role is to use software engineering best practices to support rapid and safe delivery of high-quality software.
+
+## Your Focus
+
+At this stage in your career you should focus on the success and professional development of:
+
+- your direct reports (i.e. the people who you line manage)
+- the people within your Delivery Team
+
+## Your Responsibilities
+
+Your key responsibilities are to:
+
+- Lead **System Design** activities within a Delivery Team
+- Ensure there is sufficient **Documentation** to support our products and systems
+- Attend SDMT to assist with **SDMT Operational Planning**
+- Carry out deliberate **Mentoring** to help others develop their knowledge and skills
+- Undertake your **Line Management** duties, in accordance with guidance
+
+When the above responsibilities are met, you may also:
+
+- Carry out **Feature Development** work within a Delivery Team
+- Complete **20% Time** work within each Delivery Team sprint
+- **Pair** with other members of a Delivery Team where this would be productive
+- **Buddy** with newer developers to help them settle into their role
+
+Details on how these responsibilities should be carried out can be found in The Way We Work.
+
+## How You Will Be Supported
+
+Your line manager will:
```

Comment:
Line manager: A line manager should:

or

Line manager: Your line manager is expected to:

or

Your line manager is required to: